screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -704,7 +704,7 @@ class EditTransactionViewModel @Inject constructor(
                     dateTime = when {
                         loadedTransaction().dateTime == null &&
                                 dueDate.value == null -> {
-                            timeNowUTC()
+                            getTrueDate(LocalDate.now(),timeNowLocal().toLocalTime())
```

Comment:
Hey @akashs056 how sure are you that this doesn't break other stuff? Like balances, income/expense calculation. Can you please write in Telegram for the community to test QA it? This code line was updated multiple times by multiple people resulting in different bugs (e.g. it fixes one thing but breaks another)